pyproject.toml

```diff
@@ -30,20 +29,20 @@ keywords = [
 dynamic = ['version']
 dependencies = [
     "attrs>=22.2.0",
-    "botorch>=0.8.1",
+    "botorch>=0.9.3",
     "cattrs>=23.2.0",
     "exceptiongroup",
     "funcy>=1.17",
     "gpytorch>=1.9.1",
     "ngboost>=0.3.12",
     "numpy>=1.24.1",
-    "pandas[parquet]>=1.4.2",
```

Comment:
Can you comment briefly on why this was needed and is no longer needed?

Reply:
we used to serialize via parqet but adrian changed it to custom hence we dont neet the additional dependency anymore

am currently getting an issue in 3.12 related to serialization maybe related